Makefile

```diff
@@ -664,7 +664,7 @@ dist/kubernetes.swagger.json: Makefile
 	@mkdir -p dist
 	# recurl will only fetch if the file doesn't exist, so delete it
 	rm -f $@
-	./hack/recurl.sh $@ https://raw.githubusercontent.com/kubernetes/kubernetes/v1.31.3/api/openapi-spec/swagger.json
+	./hack/recurl.sh $@ https://raw.githubusercontent.com/kubernetes/kubernetes/v1.32.2/api/openapi-spec/swagger.json
```

Comment:
Would it makes sense if this be derived from https://github.com/argoproj/argo-workflows/blob/main/hack/k8s-versions.sh?

Not necessary for this PR, but something to consider.

Reply:
I think not because thats for the CI version and we will probably update the CI and k8s libs separately